ironfish-cli/src/commands/wallet/burn.ts

```diff
@@ -48,6 +56,16 @@ export class Burn extends IronfishCommand {
         'Minimum number of block confirmations needed to include a note. Set to 0 to include all blocks.',
       required: false,
     }),
+    rawTransaction: Flags.boolean({
+      default: false,
+      description:
+        'Return raw transaction. Use it to create a transaction but not post to the network',
+    }),
+    expiration: Flags.integer({
+      char: 'e',
+      description:
+        'The block sequence after which the transaction will be removed from the mempool. Set to 0 for no expiration.',
```

Comment:
This is a consensus thing, not just a mempool eviction thing. The actual purpose of expiration is that it's "the block sequence that the transaction can not be mined after". Which is very important for finality.